processor/tsconfig.json

```diff
@@ -1,29 +1,46 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "preserveConstEnums": true,
-    "sourceMap": true,
-    "newLine": "LF",
-    "target": "es2015",
-    "moduleResolution": "node",
-    "noImplicitReturns": true,
-    "importHelpers": true,
-    "outDir": "dist",
-    "allowJs": false,
-    "noUnusedLocals":true,
-    "strict": true,
-    "declaration": true,
-    "declarationDir": "./typings"
+    /* Basic Options */
```

Comment:
Ooo I like how you split this up!

Reply:
It was copied from the `packages\@azure\template` so kudos to @mikeharder I guess?